recipes/flask-httpauth/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "Flask-HTTPAuth" %}
+{% set version = "3.2.0" %}
+{% set sha256 = "cf53acd441321ff6f369ae54e423dbec8d4c6204934c13368651cdd402805b61" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - flask
+
+  run:
+    - python
+    - flask
+
+test:
+  imports:
+    - flask_httpauth
+
+about:
+  home: http://github.com/miguelgrinberg/flask-httpauth/
+  license: MIT
```

Comment:
Can we please add a note to include a license file once we can? A link to issue ( https://github.com/miguelgrinberg/Flask-HTTPAuth/issues/45 ), would also be good.